### DISCUSSION TITLE
Question: Can I grab the camera zoom level?

### DISCUSSION THREAD

**Discussion by Georodin** [google/model-viewer#3159] [Q&A]
### Description
I want to have the Hotspots fading in and out depending on the zoom level/distance.
Where could I grab it? I briefly checked the documentation but did not find the desired field/member.
Thanks in advance!

### Version
1.10.1 newest as of 01/27/2022

**Comment by samaneh-kazemi** [comment]
Hi @Georodin, have you checked these two examples in our docs?

[Annotations](https://modelviewer.dev/examples/annotations/)
[Dimensions](https://modelviewer.dev/examples/annotations/#dimensions)

**Comment by samaneh-kazemi** [comment]
What you can do is to put a listener on [camera events](https://modelviewer.dev/docs/#stagingandcameras-events), and when [camera orbit/zoom level](https://modelviewer.dev/docs/#entrydocs-stagingandcameras-methods-getCameraOrbit) changes, change the CSS attributes for your hotspot.
